tests/test_opds_import.py

```diff
@@ -561,33 +609,48 @@ def test_import_and_make_presentation_ready(self):
     def test_status_and_message(self):
         path = os.path.join(self.resource_path, "unrecognized_identifier.opds")
         feed = open(path).read()
-        imported_editions, imported_pools, imported_works, error_messages = (
+        imported_editions, imported_pools, imported_works, failures = (
             OPDSImporter(self._db).import_from_feed(feed)
         )
 
-        [message] = error_messages.values()
-        eq_(404, message.status_code)
-        eq_("I've never heard of this work.", message.message)
+        [failure] = failures.values()
+        eq_("I've never heard of this work.", failure.exception)
 
 
-    def test_import_failure_becomes_status_message(self):
-        # Make sure that an exception during import stops the import process, 
-        # and generates a meaningful error message.
+    def test_import_edition_failure_becomes_coverage_failure(self):
+        # Make sure that an exception during import generates a
+        # meaningful error message.
 
         feed = self.content_server_mini_feed
 
-        imported_editions, imported_pools, imported_works, error_messages = (
+        imported_editions, pools, works, failures = (
             DoomedOPDSImporter(self._db).import_from_feed(feed)
         )
 
         # Only one book was imported, the other failed.
         eq_(1, len(imported_editions))
 
-        # The other failed to import, and became a StatusMessage
-        message = error_messages['http://www.gutenberg.org/ebooks/10441']
-        eq_(500, message.status_code)
-        assert "Utter failure!" in message.message
+        # The other failed to import, and became a CoverageFailure
+        failure = failures['http://www.gutenberg.org/ebooks/10441']
```

Comment:
Do a quick check to verify that this is a CoverageFailure object. Just checking `.exception` leaves it a little ambiguous. Also check `.status`.